boards/arc/emsdp/Kconfig.defconfig

```diff
@@ -23,4 +23,21 @@ config SPI_DW
 
 endif # SPI
 
+if I2C
+
+config I2C_DW
+	default y
+
+if I2C_DW
+
+config I2C_DW_CLOCK_SPEED
+	default 200
```

Comment:
Is this used? Or is it taken from dts?

Reply:
It is used. not from dts, just from board kconfig.